src/app/modules/startup/internal/user_profile_confirm_password_state.nim

```diff
@@ -24,7 +24,7 @@ method executePrimaryCommand*(self: UserProfileConfirmPasswordState, controller:
   elif self.flowType == FlowType.FirstRunNewUserNewKeycardKeys:
     controller.storeKeycardAccountAndLogin(storeToKeychain, newKeycard = true)
   elif self.flowType == FlowType.FirstRunOldUserImportSeedPhrase:
-    controller.storeImportedAccountAndLogin(storeToKeychain)
+    controller.storeImportedAccountAndLogin(storeToKeychain, recoverAccount = true)
```

Comment:
@jrainville that it went unnoticed most likely cause in the case of Mac users the last step happens in the Biometrics state which already has `recoverAccount` set to `true`.